CRM/Banking/PluginImpl/Importer/XML.php

```diff
@@ -315,7 +327,12 @@ protected function filterMatches($payment_node, $filter) {
    */
   protected function import_payment($payment_spec, $payment_node, $stmt_data, $index, $params) {
     $config = $this->_plugin_config;
-    $progress = ((float)$index / (float) $params['total_tx_count']);
+    $total_tx_count = (float) $params['total_tx_count'] ?? 1.0;
+    if ($total_tx_count) {
+      $progress = ((float)$index / ((float) $params['total_tx_count'] ?? 0.0));
```

Comment:
And if so, this should read `$total_tx_count` instead of `$params['total_tx_count']`, right?